src/ui/src/abstractlogview.cpp

```diff
@@ -2053,11 +2053,11 @@ void AbstractLogView::drawTextArea( QPaintDevice* paintDevice )
         }
 
         const auto untabifyHighlight = [ &logLine ]( const auto& match ) {
-            const auto prefix = logLine.leftRef( match.startColumn() );
+            const auto prefix = QStringView(logLine).left( match.startColumn() );
```

Comment:
I'd like to avoid hard depency on Qt 5.10 for now. Here a macro version check can be used. Ugly but allows to support more distributions